packages/protocol/contracts/bridge/libs/LibBridgeSignal.sol

```diff
@@ -8,6 +8,7 @@
 // ╱╱╰╯╰╯╰┻┻╯╰┻━━╯╰━━━┻╯╰┻━━┻━━╯
 pragma solidity ^0.8.9;
 
+
```

Comment:
```suggestion
```
Lets only leave one empty line between `pragma` and `import`, i think we can install a vscode prettier plugin ( like: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode ), make it formatting on save , then it can help doing this kind of trivial work.

Reply:
You can run `pnpm install` then use github desktop software to commit changes. Then it will automatically format all files. Do not format manually or using other tools.